export_clip.lua

```diff
@@ -38,32 +38,36 @@ function export_loop_clip()
   local b = mp.get_property_number("ab-loop-b")
   local path = mp.get_property("path")
   if a and b then
+    mp.osd_message("slow starting...")
     local outfile = slow_start()
     local cmd = {
-      'ffmpeg',
-      '-hide_banner',
-      '-ss', tostring(a),
-      '-i', path,
-      '-t', tostring(b-a),
-      '-c:v', 'libx264',
-      '-crf', '22',
-      '-pix_fmt', 'yuv420p',
-      '-filter_complex', 'scale=iw*min(1\\,min(1280/iw\\,720/ih)):-2',
-      '-an', '-sn',
-      '-map_metadata', '-1',
-      -- '-map', '0',
-      '-v', 'error',
+      "run",
```

Comment:
what is this "run" command

Reply:
https://mpv.io/manual/stable/#command-interface-[-]]]